tests/unit/test_tag_google_analytics_gtag.py

```diff
@@ -59,6 +59,28 @@ def test_identify(self):
         r = GoogleAnalyticsGTagNode().render(Context({'user': User(username='test')}))
         assert "gtag('set', {'user_id': 'test'});" in r
 
+    def test_identity_context_specific_provider(self):
+        """
+        The user_id variable must be set according to
+        google_analytics_gtag_identity variable in the context.
+        """
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'google_analytics_gtag_identity': 'foo_gtag_identity',
+            'analytical_identity': 'bar_analyitical_identity',
```

Comment:
I understand this is a test and the values can be arbitrary. Yet still, the typos are not intentional, are they?

```suggestion
            'analytical_identity': 'bar_analytical_identity',
```